restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -151,23 +152,23 @@
 	"externalServiceStepIgnoredParamsWarning": "Parámetros de servicio no se utilizan en las solicitudes de POST/PUT por defecto. Verifique 'Rellenar automáticamente de los parámetros del servicio' para alterar este comportamiento",
 	"externalServiceStepUsernameTooltip": "Nombre de usuario para la autenticación HTTP BASIC. Déjelo en blanco si no se utiliza la autenticación.",
 	"externalServiceStepUsernameLabel": "Usuario",
-	"externalServiceStepPasswordTooltip": "Contraseña para la autenticación HTTP BASIC. Déjelo en blanco si no se utiliza la autenticación.",
+	"externalServiceStepPasswordTooltip": "Contraseña para la autenticación HTTP BASIC. Dejar en blanco si no se utiliza la autenticación.",

```

Comment:
Grammar error found. Change for:
"externalServiceStepPasswordTooltip": "Contraseña para la autenticación HTTP básica. Déjelo en blanco si no se utiliza la autenticación.",